jeremy2918/tsseract
# Added Hats and Graphic Shirt

# [Avatar Items](https://trello.com/c/yXOkntsu/74-i-want-to-add-all-the-items-to-the-avatar-editor)


This PR adds all the avatar items that were missing to the avatar editor. It also makes some code improvements for readability.

## Testing Steps

- Pull and checkout to this branch.
- `npm install`
- `npm run tsc`
- `npm test` All tests should be passing successfully.
- `npm run watch`
- Check the new items in the avatar editor.
